lib/msal-common/src/app/module/AuthorizationCodeModule.ts

```diff
@@ -120,32 +122,34 @@ export class AuthorizationCodeModule extends AuthModule {
                 throw ClientAuthError.createEndpointDiscoveryIncompleteError(e);
             }
         }
-        const { tokenEndpoint } = acquireTokenAuthority;
-
-        const tokenReqParams = new ServerTokenRequestParameters(
-            this.clientConfig.auth.clientId,
-            tokenRequest,
-            codeResponse,
-            this.getRedirectUri(),
-            this.cryptoObj
-        );
-
-        const acquiredTokenResponse = await this.networkClient.sendPostRequestAsync<ServerAuthorizationTokenResponse>(
-            tokenEndpoint,
-            {
-                body: tokenReqParams.createRequestBody(),
-                headers: tokenReqParams.createRequestHeaders()
-            }
-        );
 
+        const { tokenEndpoint } = acquireTokenAuthority;
+        let tokenReqParams;
         try {
```

Comment:
same thing here--i think minimizing what's inside a `try` block is generally good.

Reply:
The only line here which would probably not throw an error is `this.account = tokenResponse.account`. Again, if there is a better way to do this, I am open to it, but here are the steps and possible errors:
1. getting a cached request - throws if issue in cache or parsing
2. creating token request parameters - throws validation errors
3. network request - throws if issue in fetch request
4. validate response - throws validation errors
5. creating token response handler - no errors thrown in constructor
6. creating token response - throws validation and parsing errors
7. account assignment - no errors thrown here

Reply:
Same thing, if most everything can realistically throw then it's probably fine.